ivy/array/manipulation.py

```diff
@@ -251,8 +251,27 @@ def stack(
         ivy.Array instance method variant of ivy.stack. This method simply
         wraps the function, and so the docstring for ivy.stack also applies
         to this method with minimal changes.
+
+        Examples
+        --------
+        >>> x = ivy.array([[2,3],[1,4]])
+        >>> y = ivy.array([[4,5],[6,7]])
+        >>> x.stack([y])
+        ivy.array([[[ 6,  8],
+                [ 7, 11]]])
+        >>>
+
+        >>> x = ivy.array([[1,2],[3,4]])
+        >>> y = ivy.array([[5,6],[7,8]])
+        >>> x.stack([y],axis=1)
+        ivy.array([[[ 6,  8]],
+            [[10, 12]]])
+        >>> x.stack([y],axis=0)
+        ivy.array([[[ 6,  8],
+                [10, 12]]])
+        >>>
         """
-        return ivy.stack([self._data] + arrays, axis=axis, out=out)
+        return ivy.stack(self._data.concat(arrays), axis=axis, out=out)
```

Comment:
should be 

```self.concat``` instead of ```self._data.concat```

Reply:
Thanks for pointing that out. I've updated my pr.

Reply:
You'll need to update the example now since the outputs are different